gc_tests/run_tests.rs

```diff
@@ -65,6 +75,10 @@ fn main() {
             let lib_arg = [CRATE_NAME, "=", lib_path.to_str().unwrap()].concat();
             let deps_arg = ["dependency=", deps_path.to_str().unwrap()].concat();
 
+            let san_flags = sanitizers.iter().map(|x| format!("-Zsanitizer={}", x));
+
+            compiler.args(san_flags);
```

Comment:
I'm not sure the blank line between the assignment and (only) use is helpful.

Reply:
Fixed: 89252e717bd6c9a5f8038ccd88df42b2017e241e